lib/trento/application/usecases/hosts/hosts.ex

```diff
@@ -36,4 +47,41 @@ defmodule Trento.Hosts do
         subscription_count
     end
   end
+
+  @spec deregister_host(Ecto.UUID.t(), DateService) :: :ok | {:error, any}
```

Comment:
Could be have something more specific for `{:error, any}`. `{:error, :host_alive} | {:error, :host_not_found}`